src/default_env_vars.yml

```diff
@@ -2,7 +2,7 @@ REDOWNLOAD_FROM_FMS: False
 USING_PICKLE: False
 DEBUG: False
 GENERATE_REPORTS: False
-ALLIANCE_RELEASE: "0.0.0"
+ALLIANCE_RELEASE: "5.3.0"
```

Comment:
These vars need to not be set and let them be sent in via ENV variables.

Reply:
Aren't they overwritten? Those are just defaults if they're not set by the environment (which they are in the GoCD pipeline).

Reply:
I just set the it to 0.0.0 to be safe.